manifests/kustomize/README.md

```diff
@@ -51,7 +50,7 @@ and open http://localhost:8080/
 
 
 
-# Uninstall
+## Uninstall
 You can uninstall Kubeflow Pipelines by running
 ```
 kubectl delete -f https://raw.githubusercontent.com/kubeflow/pipelines/$PIPELINE_VERSION/manifests/kustomize/namespaced-install.yaml
```

Comment:
`export PIPELINE_VERSION=0.1.26` should also be add within this code block. A user may not uninstall their deployment right away.

Reply:
done